packages/server/src/api/endpoints/subscription.ts

```diff
@@ -140,7 +140,7 @@ export async function subscriptionEndpoints(server: FastifyInstance): Promise<vo
         properties: {
           pricePerUnit: { type: 'number' },
           units: { type: 'number' },
-          status: { type: 'string' },
+          status: { type: 'string', enum: ['active', 'error', 'paused', 'canceled'] },
```

Comment:
@anbraten `processing` is missing in the enumeration at least when we look into the subscription entity.